styles/styles.css

```diff
@@ -0,0 +1,6 @@
+@import "variables.css";
```

Comment:
I like this import structure and keeping files separate. Very nice. You'll use this more and more as you keep on.